src/Weasel.Postgresql/Tables/IndexDefinition.cs

```diff
@@ -518,6 +518,11 @@ private static string canonicizeColumn(string expression)
             expression = expression.Substring(1, expression.Length - 2);
         }
 
+        if (expression.StartsWith('"') && expression.EndsWith('"'))
+        {
+            expression = expression.Trim('"');
+        }
```

Comment:
```suggestion
        expression = expression.Trim('"');
```
Is enough here, as if nothing needs to be trimmed, then the original string is returned.
Thus avoiding the two calls to StartsWith and EndsWith.

But on L516 above (GH doesn't allow a comment there) the literals should be changed to chars. It's more performant as the string-literals.
```diff
-while (expression.StartsWith("(") && expression.EndsWith(")"))
+while (expression.StartsWith('(') && expression.EndsWith(')'))
```

Reply:
I wanted to ensure the expression to be enclosed within double quotes to remove it, in hindsight, as you indicated, `StartsWith` and `EndsWith` are not required.

With regards to your comment on the while loop, it is an existing code so you can't comment unless it is a PR being reviewed. I will fix this as well.

Reply:
> it is an existing code so you can't comment unless it is code change in a PR being reviewed

+/- some LOCs outside of that change. More or less what GH shows you for review.
Anyway, thanks for picking that change too.